### ISSUE TITLE
Inquiry About Training Duration

### ISSUE THREAD

**Issue by tom68-ll** [Bahuia/C3#2]
I would like to inquire about the approximate time required to train a complete C3 model as per the environment and parameter settings mentioned in the paper. Based on the provided code, it seems that this might be a considerably lengthy process. Could you please provide an estimate of the training duration?

**Comment by ZSY-SZ** [comment]
Our experiments were conducted on one single NVIDIA RTX 4090, and we've provided our [training logs](https://github.com/Bahuia/C3/tree/main/train/ckpt/logs) in which you will find the detailed loss and duration of each training epoch.

**Comment by tom68-ll** [comment]
Thanks for your answer!

**Comment by tom68-ll** [comment]
Hello @ZSY-SZ , 
May I ask if the log results recorded in the code are obtained by executing the following command？
Such as in Spider-Stream:
`python train/main.py --cuda_visible_devices 0 --do_train true --do_cl_eval true --predictor_backbone_plm t5-large-lm-adapt --predictor_batch_size 1 --predictor_gradient_accumulation_steps 12 --dataset_type spider --dataset spider_perm_1 --first_task_id 0 --last_task_id 10 --task_num 11 --teacher true --teacher_plm t5-large-lm-adapt --teacher_with_context True --task_adaptation_with_teacher_logits false`
We used the same commands, but the results differ significantly from those recorded in the log. For example, on **task_1**, the best performance in the **eval_log** is Epoch 949: exec score": 0.7317073170731707, "match score": 0.6951219512195121, yet we can only achieve an "Epoch 799: exec score: 0.6097560975609756, match score: 0.5975609756097561".
Could you offer further assistance, such as suggestions on other hyperparameter settings? Thank you very much.
